src/test/ui/allocator/no_std-alloc-error-handler-custom.rs

```diff
@@ -84,6 +84,13 @@ fn panic(panic_info: &core::panic::PanicInfo) -> ! {
     }
 }
 
+// Becuse we are complying this code with `-C panic=abort`, this wouldn't normally be needed.
```

Comment:
```suggestion
// Because we are complying this code with `-C panic=abort`, this wouldn't normally be needed.
```
typo

Reply:
Oh my goodness, there are _two_ typos in there. My apologies, I had not finished my morning coffee when I typed this up.

Reply:
Fixed in ba9fe54!